src/main/kotlin/com/criptext/mail/scenes/mailbox/data/EmailInsertionSetup.kt

```diff
@@ -245,6 +245,30 @@ object  EmailInsertionSetup {
             body
     }
 
+    fun getDecryptedEmailHeaders(signalClient: SignalClient,
+                              body: String,
+                              metadata: EmailMetadata): String? {
+        if(metadata.senderRecipientId == SignalUtils.externalRecipientId
+                || metadata.isExternal == true) return null
```

Comment:
what If you call `getDecryptedEmailBody` after this special validation

Reply:
in reality this validation should not be there since it is trying to cover a mistake from backend, was just in place to test that.